app/services/matchers/fowlerwelch/model1.js

```diff
@@ -6,7 +6,7 @@ const path = require("path");
 const filenameForLogging = path.join("app", __filename.split("app")[1]);
 const { matchesHeader } = require("../../matches-header");
 
-function matchesModel(packingList, filename, regexExpression, trader) {
+function matchesModel(packingList, filename, regexExpression, _trader) {
```

Comment:
notice same issue with Mars model, if this parameter is not needed, just get rid of it. 
Or, you can just refactor to only have matches method and remove matchesModel